### PR TITLE
INGK-600 Add null character to all messages

### PR DESCRIPTION
**Pull request by MartinALL91**
### Description

Unify the length of all EoE command messages.

Fixes # INGK-600

### Type of change

- Use the null character on all command messages

### Tests
Updated tests accordingly

### CODE REVIEW

**Review by WoodElv — APPROVED** [review]

File: tests/eoe/test_eoe_network.py [hunk 1]
```diff
@@ -63,10 +63,13 @@ def test_eoe_command_msg(cmd, subnode, data, dtype):
...
-    data_field = msg[-EOE_MSG_DATA_SIZE:]
+    subnode_field = msg[EOE_MSG_CMD_SIZE:
+                        EOE_MSG_NODE_SIZE + EOE_MSG_TERMINATOR_SIZE + 1]
+    data_field = msg[-(EOE_MSG_DATA_SIZE + EOE_MSG_TERMINATOR_SIZE):]
+    null_terminator = b"\x00"
     assert len(msg) == EOE_MSG_FRAME_SIZE
     assert cmd_field == cmd.encode('utf-8')
-    assert subnode_field == f'{subnode:0{EOE_MSG_NODE_SIZE}d}'.encode('utf-8')
-    assert data_field == data_bytes + data_filling
+    assert subnode_field == f'{subnode:0{EOE_MSG_NODE_SIZE}d}'.encode('utf-8')\
```
Comment (pzinemanas) [inline tests/eoe/test_eoe_network.py:72]: I think this line was not correctly formatted (black)

**Review by pzinemanas — APPROVED** [review]

File: tests/eoe/test_eoe_network.py (hunk 1 above)
Comment (MartinALL91) [reply tests/eoe/test_eoe_network.py]: Done

File: ingenialink/eoe/network.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def _build_eoe_command_msg(cmd, node=1, data=None):
         node_field = f"{node:0{EOE_MSG_NODE_SIZE}d}".encode("utf-8")
         null_terminator = b"\x00"
```
Comment (GullinBustin) [inline ingenialink/eoe/network.py:125]: Do you think that it can be a constant? you use it more than once and in the test

Comment (MartinALL91) [reply ingenialink/eoe/network.py:125]: Yes, I will create a constant.

**Pull request merged**